src/foam/nanos/u2/navigation/NavigationView.js

```diff
@@ -6,19 +6,19 @@ foam.CLASS({
   documentation: 'Navigation bars',
 
   css: `
-    ^ {
-      position: fixed;
-      z-index: 10001;
-    }
     ^top-nav {
-      width: calc(100% - 200px);
+      width: calc(100% - 1px);
```

Comment:
I think the width should just be 100%, but ever time I put it at 100% I lose all of the items on the side menu. Using the calc is the only way that I can keep those showing up.